packages/react-native-sdk/README.md

```diff
@@ -0,0 +1,59 @@
+# @onflow/react-native-sdk
+
+A React Native library that provides hooks for interacting with the Flow blockchain. It helps you authenticate users, run Cadence scripts and transactions, listen to events, and manage network configuration directly from your components. Fully compatible with Expo.
+
+## 🔧 Installation
+
+```bash
+npm install @onflow/react-native-sdk
+```
+
+## 🧩 Quick Setup
+
+Before using any hooks, wrap your app in the `FlowProvider`:
+
+```tsx
+import {FlowProvider} from "@onflow/react-native-sdk"
+import flowJson from "../flow.json"
+
+<FlowProvider
+  config={{
+    accessNodeUrl: "https://rest-mainnet.onflow.org",
+    flowNetwork: "mainnet",
+    walletconnectProjectId: "YOUR_WALLETCONNECT_PROJECT_ID",
+    appDetailTitle: "My On Chain App",
+    appDetailIcon: "https://example.com/icon.png",
+    appDetailDescription: "A decentralized app on Flow",
+    appDetailUrl: "https://myonchainapp.com",
+  }}
+  flowJson={flowJson}
+>
+  <App />
+</FlowProvider>
+```
+
+## 🪝 Available Hooks
```

Comment:
I would put a link to a full list in case it gets out of date. And also put a note like here is a look at some of the hooks available just to account for the list becoming not up to date.